R/utils.R

```diff
@@ -67,15 +69,16 @@
     missing_prior <- utils::capture.output(print(
       prior |>
         filter(is.na(.data$prior_old)) |>
-        select(
-          prior = prior_new, dplyr::all_of(cols), source = source_new
-        )
+        select(prior = prior_new, dplyr::all_of(cols), source = source_new)
     ))
-    msg <- c(
-      "i" = "No available prior to replace in old_prior found for:",
-      missing_prior
-    )
-    cli_abort(message = msg)
+    if (warn) {
+      msg <- c(
+        "!" = "No available prior to replace in old_prior found for:",
```

Comment:
From the user perspective they are passing in prior so I think it would be better to update the internal args to respect this and the warning to match

Reply:
Yes fair enough I think there should be a way to rewrite this function to make more sense from external perspective (even though it is an internal function and makes sense as is now)